browse/static/css/arXiv.css

```diff
@@ -827,6 +827,31 @@ p.tagline {
   cursor: pointer;
 }
 
```

Comment:
add a note for future editors:
```suggestion
/*added by Papers With Code*/
```